vscode-ext/package.json

```diff
@@ -42,12 +42,14 @@
 		"watch": "tsc -watch -p ./",
 		"pretest": "npm run compile && npm run lint",
 		"lint": "eslint src --ext ts",
+		"format": "eslint src --ext ts --fix",
 		"test": "node ./out/test/runTest.js",
 		"vscode:prepublish": "npm run esbuild-base -- --minify",
 		"esbuild-base": "esbuild ./src/extension.ts --bundle --outfile=out/main.js --external:vscode --format=cjs --platform=node",
 		"esbuild": "npm run esbuild-base -- --sourcemap",
 		"esbuild-watch": "npm run esbuild-base -- --sourcemap --watch",
-		"test-compile": "tsc -p ./"
+		"test-compile": "tsc -p ./",
+		"package": "vsce package"
```

Comment:
Added `vsce` dependency and this script to generate a `vsix` for local extension testing. Is there maybe a better way or should I maybe update a dev guide with these steps?

Reply:
So VSCode lets you [debug extensions](https://code.visualstudio.com/api/get-started/your-first-extension), which I find more convenient for all but the very last phase of testing. That said, it could still be useful to have vsce package available in addition. 
And it would be great if you could add it all to the dev guide!

Reply:
Ah thanks! This was my first time working on a VSCode extension and I didn't know that. I would PR `metalbear/mirrord.dev` repo (https://github.com/metalbear-co/mirrord.dev/tree/main/content/docs/developer) for adding to dev guide?

Reply:
We discussed this on `#dev`. Consensus seems to be new page should be added to developer docs for VSCode Extension development.